database/migrations/2020_09_26_124457_add_facebook_login.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddFacebookLogin extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table(
+            "users",
+            function (Blueprint $table): void {
```

Comment:
Do we need these Facebook ids in `users` table? Maybe some `facebook_users` table would be easier to maintain? What if we would like to have seven different providers? `users` table would have seven new (and almost always nulled) columns?